docs/intro/glossary.md

```diff
@@ -0,0 +1,255 @@
+
+For example, recording a measurement of 400 from the capacity of a battery at 2022-01-01 22:00:00.000 UTC would constitute a sample **`(2022-01-01 22:00:00.000 UTC, 400)`**.
+
+#### Sample Value
+
+A [measurement](#measurement) stored in a [sample](#sample).
+
+### Time Series
+
+A sequence of [samples](#sample), often organized by [timestamp](#timestamp) and typically with regular intervals. However, irregular (event-based) time series are also possible.
+
+For example, a time series representing measurements of a battery's capacity at 2022-01-01 22:00:00.000 UTC every second for 5 seconds would appear as follows:
+
+```
+(2022-01-01 22:00:00.000 UTC, 400)
+(2022-01-01 22:00:01.000 UTC, 401)
+(2022-01-01 22:00:02.000 UTC, 403)
+(2022-01-01 22:00:03.000 UTC, 402)
+(2022-01-01 22:00:04.000 UTC, 403)
+```
+
+### Timeseries
+
+Same as [time series](#time-series).
+
+### Metric ID
+
+An identifier for a [component](#component)'s [metric](#metric), typically a string (`str`).
+
+Components belonging to the same [category](#component-category) have the same set of metric IDs.
+
+For example, the metric ID for the capacity of a [battery](#battery) is simply **`capacity`**.
+
+### Time Series ID
+
+An identifier for a [time series](#time-series) originating from a [metric](#metric) of a specific component. Typically a string (`str`) derived from the tuple **([component ID](#component-id), [metric ID](#metric-id))** for [components](#component).
+
+For example, a time series for the capacity of a battery with component ID 5 has the ID **(component_id, metric_id)** (or **`f"{component_id}_{metric_id}"`**).
+
+### Timeseries ID
+
+Same as [time series ID](#time-series-id).
+
+## Metrics
+
+### Gross Consumption
+
+Consumption before accounting for any local generation from [solar](#solar-panel), [wind](#wind-turbine) or [CHP](#chp-plant).
+
+### Net Consumption
+
+This term traditionally refers to the difference between the [gross consumption](#gross-consumption) and the local generation (like [PV](#pv) production). It is the electricity consumption that needs to be provided by the [battery](#battery) or from the main [grid](#grid).
+
+### Net Load
+
+Same as [net consumption](#net-consumption).
+
+### Residual Consumption
+
+In [microgrid](#microgrid) context sometimes used as the remaining difference between the [net consumption](#net-consumption) and the [battery](#battery) power, i.e. what we define as [grid power](#grid_power).
+
+### Residual Load
+
+Same as [residual consumption](#residual-consumption).
+
+### State of Charge
+
+The level of charge of a [battery](#battery) relative to its capacity, expressed in percentage points. Calculated as the ratio between the remaining energy in the battery at a given time and the maximum possible energy under similar health conditions. [Source](https://epicpower.es/wp-content/uploads/2020/08/AN028_SoC-SoH-SoP-definitions_v3.pdf)
```

Comment:
This whole text could move above, where `SoC` is mentioned earlier, and here we could just add a link.

Reply:
I prefer to keep acronyms just as an expansion for what they stand for.